packages/theme-dark-extension/style/variables.css

```diff
@@ -247,14 +247,23 @@ all of MD as it is not optimized for dense, information rich UIs.
   --jp-cell-editor-active-background: var(--jp-layout-color0);
   --jp-cell-editor-active-border-color: var(--jp-brand-color1);
 
-  --jp-cell-prompt-width: 90px;
+  --jp-cell-prompt-width: 64px;
   --jp-cell-prompt-font-family: 'Source Code Pro', monospace;
   --jp-cell-prompt-letter-spacing: 0px;
   --jp-cell-prompt-opacity: 1;
-  --jp-cell-prompt-not-active-opacity: 0.5;
+  --jp-cell-prompt-not-active-opacity: 1;
   --jp-cell-prompt-not-active-font-color: var(--md-grey-300);
-  --jp-cell-inprompt-font-color: var(--md-grey-50);
-  --jp-cell-outprompt-font-color: var(--md-grey-50);
+  /* --jp-cell-inprompt-font-color: var(--md-grey-50);
```

Comment:
@Zsailer can you remove the commented lines here?

Reply:
Oops! Yeah, I'll remove those now.